web/styles/compose.css

```diff
@@ -805,6 +805,41 @@ a.compose_control_button.hide {
     margin-right: 4px;
 }
 
+#send_later {
```

Comment:
This is the little menu item inside the compose box to let you enable the send later state? Seems worth a comment to spell out that detail, since we may end up making that more of a generic menu in the future.